### PR TITLE
doc(release): Prepare release notes for cloud release January 24.04

### PR DESCRIPTION
**Pull request by pkippes**
mon-34024
mon-34022
mon-34027

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [x] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md [hunk 1]
```diff
@@ -3,6 +3,68 @@ id: cloud-release-notes
...
+
+#### Centreon IT Edition Extensions
+
+- [IT edition extensions] Added EPP license control mechanism.
+
+#### Centreon MAP
+
+- [Editor] Handled meta services in output widget.
+- [Viewer] You can now perform delete and duplicate actions from the viewer.
+
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md:67]: ### Centreon IT Edition

* **More configurations options for hosts and services**;

When configuring your hosts and services, you can now configure the interval between checks both when the resource is OK and when it is needed to validate a non-OK state. You can also configure the number of checks to validate a non-ok state before triggering and alert.
Finally, you can also  assign your resources with a dedicated icon to ease readability of the interface.

* **Dashboards Beta**:

As we keep improving our new proposition for effortlessly create and share intuitive dashboards, you can now benefit of multiple improvement, new widgets and bug fixes for the Centreon Dashboards feature.

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md [hunk 2]
```diff
@@ -3,6 +3,79 @@ id: cloud-release-notes
...
+- [API] Added an endpoint to UPDATE Cloud Token.
+
+#### Centreon IT Edition Extensions
+
+- [IT edition extensions] Added EPP license control mechanism.
+
+#### Centreon MAP
+
+- [Editor] Handled meta services in output widget.
+- [Viewer] You can now perform delete and duplicate actions from the viewer.
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md:77]: PAs de more details. Seul ce que tu as appelé "highlights" doit être conservé, et le terme highlights doit aussi être supprimé

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: cloud/releases/cloud-release-notes.md [hunk 3]
```diff
@@ -3,6 +3,19 @@ id: cloud-release-notes
 title: Centreon Cloud release notes
 ---
 
+## January 22, 2024
```
Comment (nmagniezCentreon) [inline cloud/releases/cloud-release-notes.md:6]: mettre à jour la date: lundi 26 février

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md [hunk 4]
```diff
@@ -3,6 +3,19 @@ id: cloud-release-notes
 title: Centreon Cloud - notes de release
 ---
 
+## January 22, 2024
```
Comment (nmagniezCentreon) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md:6]: same same

**Review by nmagniezCentreon — CHANGES_REQUESTED** [review]

File: cloud/releases/cloud-release-notes.md (hunk 3 above)
Comment (cg-tw) [inline cloud/releases/cloud-release-notes.md:6]:
```suggestion
## February 26, 2024
```

File: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md (hunk 4 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md:6]:
```suggestion
## February 26, 2024
```

**Review by cg-tw — APPROVED** [review]

**Review by nmagniezCentreon — APPROVED** [review]

**Pull request merged**